packages/IMAPClient-Core.package/ICCommand.class/class/fetchHeadersForUIDFrom.to..st

```diff
@@ -1,5 +1,5 @@
 commands
-fetchHeadersWithUIDFrom: aBeginId to: anEndId
+fetchHeadersForUIDFrom: aBeginId to: anEndId
```

Comment:
Please make clear that this method works on a UID range.
Use `fetchFlagsFromUIDsFrom: to:` or `fetchFlagsFromUIDRangeFrom`